.gitignore

```diff
@@ -1,6 +1,8 @@
 .DS_Store
 demo
 node_modules
+dist
```

Comment:
this is required for the NPM package - the build for the latest version has to be included in the repo because it has to be published along with NPM package, so you can use either NPM package in VUE project or direct Javascript injection from this folder if you're integrating without a package manager.